docs/CLI/admin/instancetypes.md

```diff
@@ -115,10 +115,10 @@ primehub admin instancetypes update <id>
 | displayName | optional | string | display name for the instance type |
 | description | optional | string | |
 | global | optional | boolean | when an instance type is global, it could be seen for each group |
-| groups | optional | list of connected groups (dict) | please see the `connect` examples |
-| cpuLimit | required | float | the maximum vCPU quantity. For example: `1` or `1.0` means 1 vCPU and `0.5` means half of vCPU |
+| groups | optional | list of connected groups (dict) | please see the `connect` examples. default value: false |
+| cpuLimit | optional | float | the maximum vCPU quantity. For example: `1` or `1.0` means 1 vCPU and `0.5` means half of vCPU. default value: 1 |
```

Comment:
Auto-filling feature should not change a field required or not.

We could make another section to tell users some values that could be auto-filling.

------

Marke the `required` to `required*` at the original table

| field | required | type | description |
| --- | --- | --- | --- |
| cpuLimit | required* | float | the maximum vCPU quantity. For example: `1` or `1.0` means 1 vCPU and `0.5` means half of vCPU. default value: 1 |

-------

## Auto-filling Fields

Auto-filling will happen when the inputs omitted fields

| field | value | comment |
| --- | --- | --- |
| cpuLimit | 1 |  1 vCPU |
| memoryLimit | 1 |  1 GB memory |

Reply:
nice suggestion